### ISSUE TITLE
Executor wrong result

### ISSUE THREAD

**Issue by LuttyYang** [NeonXP/MathExecutor#32]
```php
var_dump($math->execute("1 + 2 * 3 / (min(1, 5) + 2 + 1)"))
```

output: 10

```php
var_dump(1 + 2 * 3 / (min(1, 5) + 2 + 1))
```

output: 2.5

**Comment by phpfui** [comment]
I have a fix for this and should have it live next week. Problem in parsing
() and functions.

Thanks for the feedback!

**Comment by phpfui** [comment]
This has been fixed in v0.6.  Please update ASAP and let me know if you are still experiencing issues.  And thanks for the feedback!
